reddit/producer.py

```diff
@@ -0,0 +1,16 @@
+import pika
+import json
+import os
+
+def publish_stock(stocks_found):
+    # Sends the found stocks to the queue  
+    connection = pika.BlockingConnection(
+    pika.ConnectionParameters(host=os.environ.get('REDDIT_RABBITMQ_HOST')))
+    channel = connection.channel()
+
+    channel.queue_declare(queue='scraped_data')
+
+    channel.basic_publish(exchange='', routing_key='scraped_data', body=json.dumps(stocks_found))
```

Comment:
Might want to make the `exchange` and `routing_key` parameters be a function argument because these values could change